tests/Storage.test.ts

```diff
@@ -329,8 +329,9 @@ function putAllBlockData ()
 /**
  * Gets one block data
  */
-function getBlockData (height : number)
+function getBlockData (height: any)
```

Comment:
Why `any` ?

Reply:
I want to process this as a string.
The data type of 'height' is  `ulong` and it is difficult to process in TypeScript. 
Therefore, it is for processing as a string.
It should use bigint to be exact, but it would be more efficient not to convert what can be treated as a string.

Reply:
I see. I am not a fan of this approach, because numbers should be stored as numbers, not string, but if it's easier in TypeScript, let's go with this.